kqueen_ui/blueprints/manager/templates/manager/overview.html

```diff
@@ -122,29 +122,45 @@ <h3>
 
 {% block extrajs %}
 <script>
+var handleError = function (elId, errorDetails) {
+  var error =
+          "<tr class='error' >" +
+          "  <td colspan='6' class='text-center'>" +
+          "    <i class='mdi mdi-alert-circle-outline' s></i> Error occured wile loading the content" +
+          "  </td>" +
+          "</tr>";
+  $(elId + " .not-event-pointer").replaceWith(error);
+  $(elId + " table").addClass("table-error");
+  $(".error td").attr("title", errorDetails);
+};
 var loadClusters = function() {
   $('#refreshClusters').addClass('disabled');
   $.get(
     "{{ url_for('manager.data_clusters', c_page=c_page, p_page=p_page) | safe }}",
     function(data) {
       if (data.response === 200) {
         $('#tableClusters').replaceWith(data.body);
-      }
-    }
-  ).then(function() {
+    } else {
+        console.log(data);
+        handleError('#tableClusters', data.body)
+      }}
+  ).always(function() {
```

Comment:
get can fail if some internal error happens (some exception is not catched), so I suggest to add `.fail()` chain as well